speech/snippets/context_classes_test.go

```diff
@@ -27,9 +27,12 @@ func TestContextClasses(t *testing.T) {
 
 	var buf bytes.Buffer
 	gcsURI := "gs://cloud-samples-data/speech/commercial_mono.wav"
-	want := "Alternative"
 	err := contextClasses(&buf, gcsURI)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
```suggestion
		t.Fatalf("contextClasses got err: %v", err)
```

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures